doc/error_handling.md

```diff
@@ -1,5 +1,138 @@
 # Error handling
 
-You are reading a draft of the next documentation and it's in continuos update so
-if you don't find what you need please refer to:
-[C++ wrapper classes for the ABI-stable C APIs for Node.js](https://nodejs.github.io/node-addon-api/)
```

Comment:
What is our feeling about this?  I've seen some of the documentation updates leave it, but maybe it's the right time to start removing these designations.

/cc @mhdawson

Reply:
That was my thought, once we fill in the section lets remove the comment.

Reply:
Replied above at this point I'd say take it out.